linkerd/app/outbound/src/tcp/logical.rs

```diff
@@ -127,3 +110,223 @@ where
         }
     }
 }
+
+#[allow(unused_imports)]
```

Comment:
can we clean up imports here before merging? presumably this was just a temporary development thing?